src/ai/backend/common/contexts/request_id.py

```diff
@@ -0,0 +1,36 @@
+import uuid
+from contextlib import contextmanager
+from contextvars import ContextVar
+from typing import Iterator, Optional
+
+request_id_var: ContextVar[str] = ContextVar("request_id")
```

Comment:
Is there a reason why `request_id` is of type `str` instead of `UUID`?

Reply:
Thanks. I need to replace it with `UUID`.

Reply:
Oh, I left this part as `str` because of the Custom request ID.

Reply:
I just make `request_id_var` private.